### ISSUE TITLE
Implement consistent exemplars of modern telemetry signal generators with podtato-head

### ISSUE THREAD

**Issue by AlexsJones** [cncf/podtato-head#64]
In order to maximise the value of podtato-head for those learning from the exemplar implementations.
Then it is also appropriate to include telemetry signal generators. 

These should include but are not limited to:
- Logs
- Traces
- Metrics

**Comment by joshgav** [comment]
I think it would be ideal to add the [OpenTelemetry collector](https://github.com/open-telemetry/opentelemetry-collector) as a sidecar in each pod, or a daemonset in the cluster.

We could then demonstrate the collector exporting to open and proprietary backend analysis services like Jaeger. Vendors might build on this and demo connections to their own backends.

**Comment by AlexsJones** [comment]
I agree it is important to show tooling available openly without any vendor influence. 
To that end I think we could do the following:
- Introduce structured logging through some open library ( zap/klog/logrus) and print to stdout
- Use Opentracing libraries to produce spans to the agent sidecar you mentioned.
- Enable the Prometheus metrics endpoint by adding a service monitor and some simple rules in the configuration manifests.
